pkg/cmd/server/api/v1/types_test.go

```diff
@@ -51,7 +51,8 @@ masterKubeConfig: ""
 networkConfig:
   mtu: 0
   networkPluginName: ""
-nodeIP: ""
+  podTrafficNodeIP: ""
+  masterTrafficNodeIP: ""
```

Comment:
Why do we need an explicit "master traffic IP"? Shouldn't that just be what the default "node IP" is?

Reply:
self-reported node addresses are problematic... any attempts to increase use of those should be agreed on upstream